rcl/src/rcl/time.c

```diff
@@ -366,6 +363,10 @@ rcl_set_ros_time_override(
   }
   rcl_time_jump_t time_jump;
   rcl_ros_clock_storage_t * storage = (rcl_ros_clock_storage_t *)clock->data;
+  if (!storage) {
```

Comment:
Could this if statement just be a `RCL_CHECK_ARGUMENT_FOR_NULL` at the top of the function?

Reply:
Done in https://github.com/ros2/rcl/pull/685/commits/012c120bad0e56a8da83e53ec07d4b3a1c635b60.